video_xblock/static/html/youtube.html

```diff
@@ -2,7 +2,13 @@
   id="{{ video_player_id }}"
   class="video-js vjs-default-skin"
   controls
-  width="829" height="511"
+  height="560px"
   data-setup='{{ data_setup }}'
 >
+<!-- height="560" it's height of iframe minus height of control bar -->
+  {% for transcript in player_state.transcripts %}
```

Comment:
if the following lines are the same for all backends, does it make sense to move them to the separate template and incude it here?